coalib/bearlib/aspects/Spelling.py

```diff
@@ -33,3 +33,27 @@ class docs:
         ``aspects`` or ``aspect`` are always written with all lower-case
         letters or the term ``aspectsYEAH`` must match the exact expression.
         """
+
+
+@Spelling.subaspect
+class coalaCorrect:
+    """
+    This aspect dictates that the term ``coala`` must always be written with
+    with a lowercase `c`.
```

Comment:
@Asnelchristian @sils Hmm... Seems like it must not be **all** lowercase... ok :)